dependencies.properties

```diff
@@ -7,7 +7,7 @@
 # Target workspace name: com_google_api_codegen
 
 # Versions only, for dependencies which actual artifacts differ between Bazel and Gradle
-version.com_google_protobuf=3.6.1
+version.com_google_protobuf=3.7.0
```

Comment:
Can we depend on 3.7.1 instead? Protobuf 3.7.0 had some bugs for PHP. Even if gapic-generator does not really care about this it would be good to have the protobuf version consistent across our repositories. Also this version will be used by bazel build not only to build gapic-genrator itself but also to generate the actual stubs in each language (this is where PHP problems become relevant).

Reply:
oh yes